assets/app/actions/siteActions.js

```diff
@@ -55,6 +55,23 @@ export default {
     }).catch(error => alertActions.httpError(error.message));
   },
 
+  uploadFile(site, path, fileData, message) {
```

Comment:
We can move the redirect action out of the createCommit in a future PR. This one is already pretty large and that functionality had been in the action for a few weeks prior to this code being added